### PR TITLE
Port MYFACES-4679 to 2.2

### LINKED ISSUE #29648: Fix MYFACES-4679 / PH63238 in JSF (Ajax Invokes Actions Unintentionally) in JSF 2.2
**Describe the bug**  
Ajax Events Can Trigger Button Actions Unintentionally 

**Steps to Reproduce**  
If some element is ajax-enabled and specifies an event, it may trigger the action of the encompassing element. 

For example, the blur event would trigger the bean.confirm action: 
```
<h:commandButton id="confirm2" value="listener" action="#{bean.confirm()}" >
 <f:ajax event="blur" listener="#{bean.listener()}"/> 
</h:commandButton>
```

**Expected behavior**  
The ajax enabled tag should not invoke any actions other than the listener, if specified. 

**Diagnostic information:**  
 - OpenLiberty Version: Since 24.0.0.1 
 - Affected feature(s): jsf-2.2 (jsf-2.3, faces-3.0, faces-4.0 also affected but will be fixed later)
 - Java Version: N/A
 - server.xml configuration (WITHOUT sensitive information like passwords)
 - If it would be useful, upload the messages.log file found in `$WLP_OUTPUT_DIR/messages.log`

**Additional context**

### PR DESCRIPTION
**Pull request by volosied**
- [x] I have considered the risk of behavior change or other zero migration impact (https://github.com/OpenLiberty/open-liberty/wiki/Behavior-Changes).
- [x] If this PR fixes an Issue, the description includes "Fixes #FILLMEIN" or "Resolves #FILLMEIN" (verify `release bug` label if applicable: https://github.com/OpenLiberty/open-liberty/wiki/Open-Liberty-Conventions).
- [x] If this PR resolves an external Known Issue (including APARS), the description includes "Fixes #FILLMEIN" or "Resolves #FILLMEIN".

fixes #29648 (2.3+ will be fixed via PR 30276) 

ID Review is NOT needed as I'm only removing copyrights from the messages files (to match upstream since really shouldn't be licensed under EPL 2.0)

### CODE REVIEW

**Comment by volosied** [comment]
#build

**Comment by dmuelle** [comment]
Hi @volosied - are there new messages here that need ID review? If not you can add the `ID review not needed` label to bypass review. I do notice though that `Messages.js` looks odd, the metadata us being deleted and all the message file content appears to be on a single line

**Comment by dmuelle** [comment]
sorry, just noticed this is still in draft, feel free to disregard my comment if not ready

**Comment by volosied** [comment]
#build #spawn.fullfat.buckets=com.ibm.ws.jsf_fat_lWAS_faces3.0,com.ibm.ws.jsf_fat_lWAS_faces4.0,com.ibm.ws.jsf_fat_lWAS,com.ibm.ws.jsf.2.2_fat,com.ibm.ws.jsf.2.2_fat.2,com.ibm.ws.jsf.2.3_fat

**Comment by volosied** [comment]
#build #spawn.fullfat.buckets=com.ibm.ws.jsf_fat_lWAS_faces3.0,com.ibm.ws.jsf_fat_lWAS_faces4.0,com.ibm.ws.jsf_fat_lWAS,com.ibm.ws.jsf.2.2_fat,com.ibm.ws.jsf.2.2_fat.2,com.ibm.ws.jsf.2.3_fat

**Comment by volosied** [comment]
Build: https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_gDTIAKjjEe-Dk-P4EefJ2g

File: dev/com.ibm.ws.jsf.2.2_fat.2/.classpath [hunk 1]
```diff
@@ -33,6 +33,7 @@
     <classpathentry kind="src" path="test-applications/ViewScopeLeak.war/src"/>
     <classpathentry kind="src" path="test-applications/MYFACES-4512.war/src"/>
     <classpathentry kind="src" path="test-applications/PH55398.war/src"/>
+    <classpathentry kind="src" path="test-applications/PH62753.war/src"/>
```
Comment (pnicolucci) [inline dev/com.ibm.ws.jsf.2.2_fat.2/.classpath]: Wrong APAR number used.

File: dev/com.ibm.ws.jsf.2.2_fat.2/test-applications/PH63238.war/src/com/ibm/ws/jsf22/fat/PH63238/bean/TestBean.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
+/*******************************************************************************
+ * Copyright (c) 2023 IBM Corporation and others.
```
Comment (pnicolucci) [inline dev/com.ibm.ws.jsf.2.2_fat.2/test-applications/PH63238.war/src/com/ibm/ws/jsf22/fat/PH63238/bean/TestBean.java]: Is this copyright correct?

File: dev/com.ibm.ws.jsf.2.2_fat.2/test-applications/PH63238.war/resources/result.xhtml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ * Copyright (c) 2023 IBM Corporation and others.
```
Comment (pnicolucci) [inline dev/com.ibm.ws.jsf.2.2_fat.2/test-applications/PH63238.war/resources/result.xhtml]: Is 2023 correct?

**Review by pnicolucci — DISMISSED** [review]

**Review by pnicolucci — APPROVED** [review]

**Comment by pnicolucci** [comment]
#libby

**Comment by volosied** [comment]
New Build: https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_phD3EKzBEe-5Tq7vYMk6IA

**Pull request merged**